### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by ReyAdrian520**
Corrected my Calculator files to the correct path

### CODE REVIEW

File: calculator/calculator_test.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package calculator_test
+
+import (
+	"testing"
+
+	"github.com/ReyAdrian520/PivotTech/calculator"
+)
+
+func TestAdd(t *testing.T) {
```
Comment (jboursiquot) [inline calculator/calculator_test.go:9]: Use table-driven testing.

File: calculator/calculator_test.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	got := calculator.Add(2, 1)
+	expected := 3
+	if got != expected {
+		t.Errorf("not expected results, got: %v expected: %v", got, expected)
+	}
+}
+
+func TestSub(t *testing.T) {
```
Comment (jboursiquot) [inline calculator/calculator_test.go:17]: Use table-driven testing.

File: calculator/calculator_test.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	got := calculator.Sub(2, 1)
+	expected := 1
+	if got != expected {
+		t.Errorf("not expected results, got: %v expected: %v", got, expected)
+	}
+}
+
+func TestMuti(t *testing.T) {
```
Comment (jboursiquot) [inline calculator/calculator_test.go:25]: Use table-driven testing.

File: calculator/calculator_test.go [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+	got := calculator.Muti(2, 1)
+	expected := 2
+	if got != expected {
+		t.Errorf("not expected results, got: %v expected: %v", got, expected)
+	}
+}
+func TestDiv(t *testing.T) {
```
Comment (jboursiquot) [inline calculator/calculator_test.go:32]: Use table-driven testing.
You also need to test the divide-by-zero case.

**Review by jboursiquot — CHANGES_REQUESTED** [review]
Also use better branch names in the future. `new-branch` does little to convey the work taking place in the branch.
